frontend/src/tabs/authorship.pug

```diff
@@ -1,5 +1,6 @@
 #authorship
-  span.large-font Code Panel
+  .panel-title
```

Comment:
this introduces a `div` which shifts the `hide/expand all file details` downwards

Reply:
Solved.